node/node.go

```diff
@@ -102,16 +102,15 @@ func (n *Node) Start(ctx context.Context) error {
 
 	// TODO(@Wondertan): Print useful information about the node:
 	//  * API address
-	//  * Host listening address
 	log.Infof("started %s Node", n.Type)
 
 	addrs, err := peer.AddrInfoToP2pAddrs(host.InfoFromHost(n.Host))
 	if err != nil {
-		log.Errorf("Error while retreiving multiaddress information: %s", err)
+		log.Errorw("Retrieving multiaddress information", "err", err)
 		return err
 	}
 	for _, addr := range addrs {
-		log.Infow("node P2P info", "multiaddress", addr.String())
+		fmt.Println("node P2P info", "multiaddress", addr.String())
```

Comment:
why `fmt.Println`?

Reply:
Because this information needs to be printed out even if logs are turned off

Reply:
@Bidon15, Can you make it like: 
```
The p2p Host is listening on:
* Addr
* Addr
```

Reply:
Users can dump to file anything they want from stdout.

Reply:
From UX standpoint, I don't thing we need to scary light-node user with tonnes of logs, so by default I would prefer turning them off, but when you start something you need to get a sign that everything works fine and showing a succint information about the address a node listens to - makes sense.